src/main/java/seedu/clinic/logic/commands/EditCommand.java

```diff
@@ -41,19 +41,22 @@
 
     public static final String COMMAND_WORD = "edit";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Edits the details of the supplier/warehouse"
-            + " identified by the index number used in the displayed supplier/warehouse list. "
-            + "Existing values will be overwritten by the input values.\n"
-            + "Parameters: ct/TYPE i/INDEX "
+    public static final String MESSAGE_USAGE = "Edits a supplier or warehouse at the specified index."
+            + " For suppliers, only its name, phone, email and remarks can be edited."
```

Comment:
Should I include this in AddCommand as well? Since it's the first touch for users

Reply:
Ahh I might change to to something like the one in DeleteCommand!